core/templates/dev/head/css/oppia.css

```diff
@@ -626,14 +676,8 @@ textarea {
   margin-bottom: 10px;
 }
 
-.oppia-sidebar-footer a {
-  display: inline;
-  padding: 0 10px 0 0;
-  margin-top: 5px;
-}
-
-.oppia-footer-container a {
-  color: #fff;
+.d-xl-inline-flex {
```

Comment:
This seems like an override. Put all the overrides in the same place please. If needed, you can use comments to delineate the sections of the CSS file.

Reply:
Yes, moved to overrides and added comments Thanks!